layersvt/device_simulation.cpp

```diff
@@ -2101,6 +2961,207 @@ VKAPI_ATTR VkResult VKAPI_CALL GetPhysicalDeviceToolPropertiesEXT(VkPhysicalDevi
     return result;
 }
 
+#define TRANSFER_VALUE(name) dest->name = src->name
+
+// VK_VULKAN_1_1
+
+// Properties
+void TransferValue(VkPhysicalDeviceVulkan11Properties *dest, VkPhysicalDevicePointClippingPropertiesKHR *src) {
```

Comment:
Since `dest` and `src` are assumed to be non-null, use references?

Reply:
Good point.